File: 황창식/C_sharp/Class_practice_20210215/practice_2/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace practice_2
{
    class Car
    {
        private string model;
        private string color;
        private int year;
        private string company;

        public Car(string model, string color, int year, string company)
        {
            this.model = model;
            this.color = color;
            this.year = year;
            this.company = company;
        }

        public string Model { get => model; set => model = value; }
        public string Color { get => color; set => color = value; }
        public int Year { get => year; set => year = value; }
        public string Company { get => company; set => company = value; }

        public void printCarInfo()
        {
            //Console.WriteLine("----- 차량정보 -----");
            Console.WriteLine("모델명 : " + model);
            Console.WriteLine("색상 : " + color);
            Console.WriteLine("연식 : " + year);
            Console.WriteLine("제조사 : " + company);
        }
    }
}

File: 강병수/Java/210119/src/com/kbs/carApp/Car/Car.java
package com.kbs.carApp.Car;

public class Car {

	private String model;
	private String color;
	private int year;
	private String company;
	private static Car inst;
	
	public static Car getInstance(String model, String color, String company, int year) {
		if (inst == null) {
			inst = new Car(model, color, year, company);
		}
		return inst;
	}
	
	
	public Car(String model, String color, int year, String company) {
		this.model = model;
		this.color = color;
		this.year = year;
		this.company = company;
	}
	
	public String getModel() {
		return model;
	}
	public void setModel(String model) {
		this.model = model;
	}
	public String getColor() {
		return color;
	}
	public void setColor(String color) {
		this.color = color;
	}
	public int getYear() {
		return year;
	}
	public void setYear(int year) {
		this.year = year;
	}
	public String getCompany() {
		return company;
	}
	public void setCompany(String company) {
		this.company = company;
	}
	
	public void printCarInfo() {
		System.out.println("모델: " + model);
		System.out.println("색상: " + color);
		System.out.println("연식: " + year);
		System.out.println("제조사: " + company);
	}
	
	
	
	
}
